portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,34 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Jontae Greene's Portfolio</h1>
+      <p>Welcome to my portfolio!</p>
+      <!---Style to the side-->
+      <nav id="navExternal">
+          <a href="https://github.com/Corramel"><img class="externalIcon"> GitHub</a>
```

Comment:
JFYI - If you ever want a link to open in a new tab rather than the current one, you can add the target attribute with value _blank. You don't have to change it, just wanted to make sure you're aware.

Like so:
`<a href="https://github.com/Corramel" target="_blank">`

Reply:
Also, this image does not have a `src` and the class `externalIcon` doesn't appear to exist in your CSS. Is this just a placeholder?

Reply:
That's pretty useful, I didn't know that. Thanks! 
Yeah the image is a placeholder for now, I'm going to do a little image editing before adding it in.